pkg/collector/client.go

```diff
@@ -62,7 +108,37 @@ func NewClientCollector(c *api.Client) *clientCollector {
 		),
 
 		omadaClientSignalDbm: prometheus.NewDesc("omada_client_signal_dbm",
-			"The signal level for the wireless client in dBm.",
+			"The noise level for the wireless client in dBm.",
+			[]string{"client", "vendor", "ip", "mac", "ap_name", "site", "site_id", "ssid", "wifi_mode"},
+			nil,
+		),
+
+		omadaClientRssiDbm: prometheus.NewDesc("omada_client_rssi_dbm",
+			"The RSSI for the wireless client in dBm.",
+			[]string{"client", "vendor", "ip", "mac", "ap_name", "site", "site_id", "ssid", "wifi_mode"},
+			nil,
+		),
+
+		omadaClientTrafficDown: prometheus.NewDesc("omada_client_traffic_down",
```

Comment:
We should rename `omada_client_traffic_down` to `omada_client_traffic_down_bytes` so it's suffixed with the metric type